editors/code/src/main.ts

```diff
@@ -11,6 +11,9 @@ const RUST_PROJECT_CONTEXT_NAME = "inRustProject";
 
 export interface RustAnalyzerExtensionApi {
```

Comment:
Can we put a comment on this that the API exposed here is not to be assumed stable or similar?